manifests/init.pp

```diff
@@ -188,10 +188,13 @@
 #   where all networkd files are placed in
 #
 # @param manage_accounting
-#   when enabled, the different accounting options (network traffic, IO, CPU util...) are enabled for units
+#   When enabled, the different accounting options (network traffic, IO,
+#   CPU util...) are enabled for units.
 #
 # @param accounting
-#   Hash of the different accounting options. This highly depends on the used systemd version. The module provides sane defaults per operating system using Hiera.
+#   Hash of the different accounting options. This highly depends on the used
+#   systemd version. The module provides sane defaults per operating system
```

Comment:
I think if we drop all EoL OSes, the remaining OSes all use the same accounting options.

Reply:
That's what I hope for also.. but it should be done in another PR I guess